### DISCUSSION TITLE
No module named simnibs

### DISCUSSION THREAD

**Discussion by FrancescoLomi** [simnibs/simnibs#294] [9. Results and analysis]
Hi everybody, 
I'm trying to launch the python script for the ROI analysis on my ubuntu software, but the system does not find the simnibs module. Anyone knows how to solve this issue?
Thanks 

[
![Schermata del 2024-02-29 18-13-15](https://github.com/simnibs/simnibs/assets/118944911/ed789859-823d-48aa-847d-f366e34fdd8b)
](url)

**Comment by oulap** [comment]
You can't pip install simnibs directly (because the package is too big for pip), but if you have simnibs installed already you should have access to the simnibs_python interpreter. So instead of launching python by calling `python3` try calling `simnibs_python`. You should be able to import simnibs after that.
